src/util/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface Ikeys {
+  [key: string]: string | {
+    [key: string]: boolean
+  };
+}
+
+export interface Iflags {
+  version: void;
+  help: void;
+  force: boolean;
+  'generate-connections': boolean;
+}
```

Comment:
I'm not sure if we need this type. If we do, it could be named `CLIFlags` or `SyncDbOptions`.